hunts/src/solveStateFilter.js

```diff
@@ -0,0 +1,47 @@
+export const SOLVE_STATE_FILTER_OPTIONS = {
+  ALL: 0,
+  PRIORITY: 1,
+  UNSOLVED: 2,
+};
+
+function isUnsolved(tableRow) {
+  return tableRow.values.status !== "SOLVED";
+}
+
+function isSolved(tableRow) {
+  return !isUnsolved(tableRow);
+}
+
+function hasUnsolvedChildren(tableRow) {
+  return tableRow.subRows.filter(isUnsolved).length !== 0;
+}
+
```

Comment:
I think we need to do this recursively / traverse all the descendants; we care about if any descendant is unsolved to give the full context.

Reply:
good catch, fixed